### PR TITLE
Add new style for Forestry at TU Dresden

### PR DESCRIPTION
**Pull request by jmiba**

### CODE REVIEW

**Comment by jmiba** [comment]
This is also meant to be an update of another style

File: technische-universitat-dresden-forstwissenschaft.csl [hunk 1]
```diff
@@ -0,0 +1,428 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="de-DE">
+  <info>
+    <title>Technische Universität Dresden - Forstwissenschaft (author-date, German)</title>
+    <title-short>TU Dresden - Forst</title-short>
+    <id>http://www.zotero.org/styles/technische-universitat-dresden-forstwissenschaft</id>
+    <link href="http://www.zotero.org/styles/technische-universitat-dresden-forstwissenschaft" rel="self"/>
+    <link href="http://www.zotero.org/styles/die-bachelorarbeit-samac-et-al-note" rel="template"/>
+    <link href="http://www.amazon.de/Bachelorarbeit-P%C3%A4dagogischen-Hochschule-wissenschaftlicher-Lehrerbildung/dp/3708901886" rel="documentation"/>
```
Comment (adam3smith) [inline technische-universitat-dresden-forstwissenschaft.csl]: Could we get a link to documentation for this style? Ideally on the Lehrstuhl's homepage, but also OK to upload to this pull request and then put the link to the github comment in here.

File: technische-universitat-dresden-forstwissenschaft.csl [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+    <author>
+      <email>jens.mittelbach@slub-dresden.de</email>
+      <uri>https://www.jensmittelbach.de</uri>
+      <name>Jens Mittelbach</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
```
Comment (adam3smith) [inline technische-universitat-dresden-forstwissenschaft.csl]: fix both the field (engineering might be best?) and the summary please (OK to just delete the summary if you have nothing to add beyond documentation)

File: technische-universitat-dresden-historische-musikwissenschaft-note.csl [hunk 3]
```diff
@@ -105,32 +107,34 @@
...
       <if variable="container-author">
-        <names variable="container-author" suffix=", ">
-          <name and="text" delimiter-precedes-last="never" initialize="false" name-as-sort-order="first">
-            <name-part name="family"/>
-          </name>
-          <et-al term="et-al"/>
-        </names>
-        <text macro="container-title" font-style="normal"/>
-        <text macro="secondary-container-contributors"/>
+        <group delimiter=", ">
```
Comment (adam3smith) [inline technische-universitat-dresden-historische-musikwissenschaft-note.csl]: I'm a bit confused -- this shouldn't change anything. Where is the comma intended to go?

File: technische-universitat-dresden-historische-musikwissenschaft-note.csl [hunk 4]
```diff
@@ -464,7 +466,7 @@
             <text macro="contributors-long"/>
             <text macro="title-long"/>
             <group>
-              <text macro="container"/>
+              <text macro="container" suffix=", "/>
```
Comment (adam3smith) [inline technische-universitat-dresden-historische-musikwissenschaft-note.csl]: Probably better as a delimiter in the group.

File: technische-universitat-dresden-historische-musikwissenschaft-note.csl [hunk 5]
```diff
@@ -502,7 +504,7 @@
         <text macro="contributors-bib"/>
         <text macro="title-long"/>
         <group>
-          <text macro="container"/>
+          <text macro="container" suffix=", "/>
```
Comment (adam3smith) [inline technische-universitat-dresden-historische-musikwissenschaft-note.csl]: Probably better as a delimiter in the group.

File: technische-universitat-dresden-historische-musikwissenschaft-note.csl (hunk 3 above)
Comment (jmiba) [reply technische-universitat-dresden-historische-musikwissenschaft-note.csl]: This is to take care of the problem of a double comma if a contianer author is present and no secondary container contributor(s) (as in author anthologies)

File: technische-universitat-dresden-historische-musikwissenschaft-note.csl (hunk 4 above)
Comment (jmiba) [reply technische-universitat-dresden-historische-musikwissenschaft-note.csl]: You are right

File: technische-universitat-dresden-historische-musikwissenschaft-note.csl (hunk 5 above)
Comment (jmiba) [reply technische-universitat-dresden-historische-musikwissenschaft-note.csl]: You are right

File: technische-universitat-dresden-historische-musikwissenschaft-note.csl (hunk 3 above)
Comment (adam3smith) [reply technische-universitat-dresden-historische-musikwissenschaft-note.csl]: never mind, I misread the code. Makes sense.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**